### PR TITLE
[script.skin.info.service] v1.0.7

### PR DESCRIPTION
**Pull request by mikesilvo164**
### Description
- Add Album discart for Artists
- Add set details support for Home Window

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.skin.info.service/Utils.py [hunk 1]
```diff
@@ -143,24 +128,29 @@ def set_artist_properties(audio):
...
             firstyear = item['year']
         count += 1
-    if firstyear > 0 and latestyear < 2020:
+    if firstyear > 0 and latestyear < 2030:
         HOME.setProperty('SkinInfo.Artist.Albums.Newest', str(latestyear))
         HOME.setProperty('SkinInfo.Artist.Albums.Oldest', str(firstyear))
     HOME.setProperty('SkinInfo.Artist.Albums.Count', str(audio['result']['limits']['total']))
     HOME.setProperty('SkinInfo.Artist.Albums.Playcount', str(playcount))
 
+    if ADDON.getSetting("enable_debug_json") == "true":
```
Comment (Rechi) [inline script.skin.info.service/Utils.py]: use ` xbmcaddon.Addon([id]).getSettingBool(id)`, https://codedocs.xyz/xbmc/xbmc/group__python__xbmcaddon.html#gaf0205dd5b96354f30a39fbf35599e681

File: script.skin.info.service/Utils.py [hunk 2]
```diff
@@ -183,6 +173,9 @@ def set_album_properties(json_response):
     HOME.setProperty('SkinInfo.Album.Songs.Tracklist', tracklist)
     HOME.setProperty('SkinInfo.Album.Songs.Count', str(json_response['result']['limits']['total']))
 
+    if ADDON.getSetting("enable_debug_json") == "true":
```
Comment (Rechi) [inline script.skin.info.service/Utils.py]: use ` xbmcaddon.Addon([id]).getSettingBool(id)`, https://codedocs.xyz/xbmc/xbmc/group__python__xbmcaddon.html#gaf0205dd5b96354f30a39fbf35599e681

File: script.skin.info.service/Utils.py [hunk 3]
```diff
@@ -247,6 +243,9 @@ def set_movie_properties(json_response):
     HOME.setProperty('SkinInfo.Set.Movies.Years', " / ".join(years))
     HOME.setProperty('SkinInfo.Set.Movies.Count', str(json_response['result']['setdetails']['limits']['total']))
 
+    if ADDON.getSetting("enable_debug_json") == "true":
```
Comment (Rechi) [inline script.skin.info.service/Utils.py]: use ` xbmcaddon.Addon([id]).getSettingBool(id)`, https://codedocs.xyz/xbmc/xbmc/group__python__xbmcaddon.html#gaf0205dd5b96354f30a39fbf35599e681

File: script.skin.info.service/Utils.py [hunk 4]
```diff
@@ -301,17 +302,37 @@ def clear_properties():
...
+#             if debug:
+#                 log("%s%s.%i = %s" % (prefix, name, count + 1, str(result)))
+#             for (key, value) in result.iteritems():
+#                 HOME.setProperty('SkinInfo.%s%s.%i.%s' % (prefix, name, count + 1, str(key)), unicode(value))
+#                 if debug:
+#                     log('%s%s.%i.%s --> ' % (prefix, name, count + 1, str(key)) + unicode(value))
+#         HOME.setProperty('SkinInfo.%s%s.Count' % (prefix, name), str(len(data)))
+#     else:
+#         HOME.setProperty('SkinInfo.%s%s.Count' % (prefix, name), '0')
+#         log("%s%s.Count = None" % (prefix, name))
```
Comment (Rechi) [inline script.skin.info.service/Utils.py]: It's not my project, but doesn't your version control system provide you all previous versions of the code?

File: script.skin.info.service/Utils.py (hunk 1 above)
Comment (mikesilvo164) [reply script.skin.info.service/Utils.py]: @Rechi Shouldn't changing it to the below code work?
```if ADDON.getSettingBool("enable_debug_json") == "true":```


It doesn't work for me, I must be missing something obvious here. 
Any pointers would be most appreciated.

Thanks 😃

Comment (Rechi) [reply script.skin.info.service/Utils.py]: `if ADDON.getSettingBool("enable_debug_json"):`

Comment (mikesilvo164) [reply script.skin.info.service/Utils.py]: Of course... Should have tried that. Thanks!
I will update it and cleanup the commented out code and do a force push in the next few min.

**Pull request merged**